app/code/Bulbulatory/Recommendations/Helper/ConfigHelper.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+
+namespace Bulbulatory\Recommendations\Helper;
+
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Framework\App\Helper\Context;
+use Magento\Framework\Exception\NoSuchEntityException;
+use Magento\Store\Model\ScopeInterface;
+use Magento\Store\Model\StoreManagerInterface;
+
+/**
+ * Class ConfigHelper
+ * @package Bulbulatory\Recommendations\Helper
+ */
+class ConfigHelper extends AbstractHelper
```

Comment:
Plik jest w namespace `...\Helper` więc oczywiste jest, że jest helperem, nie trzeba tego już dodawać do nazwy klasy. Zwykłe `Config` wystarczy, bo namespace i tak wskazuje, czym to jest - `\Bulbulatory\Recommendations\Helper\Config`